src/rgw/rgw_reshard.h

```diff
@@ -5,26 +5,41 @@
 #define RGW_RESHARD_H
 
 #include <vector>
+#include <functional>
+
 #include "include/rados/librados.hpp"
+#include "common/ceph_time.h"
 #include "cls/rgw/cls_rgw_types.h"
 #include "cls/lock/cls_lock_client.h"
 #include "rgw_bucket.h"
 
+
 class CephContext;
 class RGWRados;
 
-
 class RGWBucketReshard {
+public:
+
   friend class RGWReshard;
 
+  using Clock = ceph::coarse_mono_clock;
```

Comment:
hey, it is

Reply:
Moved the other one of these from the function to the RGWReshard class.